server.go

```diff
@@ -493,6 +493,37 @@ func CompressHandlerLevel(h RequestHandler, level int) RequestHandler {
 	}
 }
 
+// CompressHandlerBrotliLevel returns RequestHandler that transparently compresses
+// response body generated by h if the request contains a 'br', 'gzip' or 'deflate'
+// 'Accept-Encoding' header.
+//
+// brotliLevel is the desired compression level for brotli.
+//
+//     * CompressBrotliNoCompression
+//     * CompressBrotliBestSpeed
+//     * CompressBrotliBestCompression
+//     * CompressBrotliDefaultCompression
+//
+// otherLevel is the desired compression level for gzip and deflate.
+//
+//     * CompressNoCompression
+//     * CompressBestSpeed
+//     * CompressBestCompression
+//     * CompressDefaultCompression
+//     * CompressHuffmanOnly
+func CompressHandlerBrotliLevel(h RequestHandler, brotliLevel, otherLevel int) RequestHandler {
```

Comment:
I tried to find a shorter, and cleaner option for this, but still can't find a better way.
what I see here is that now, you would have to write
```
h = fasthttp.CompressHandlerBrotliLevel(h, fasthttp.CompressBrotliDefaultCompression, fasthttp.CompressDefaultCompression)
```
or
```
h = fasthttp.CompressHandlerBrotliLevel(h, 4, 6)
```
first is barely readable, and the second isn't obvious without IDE reminding you the order. maybe, you have some ideas on that?

otherwise, this looks good to me

Reply:
How is the first not readable? It's very readable as it spells our exactly what it is. Yeah the line is a bit long but who works in an editor with 80 characters max with these days...
Or if you think it's too long you can just write it like
```go
h = fasthttp.CompressHandlerBrotliLevel(
    h,
    fasthttp.CompressBrotliDefaultCompression,
    fasthttp.CompressDefaultCompression,
)
```

Reply:
I like to comment if the ”compress...” is not need since it’s understood that Brotli is a compressed format.

Reply:
> I like to comment if the ”Compress...” is not need since it’s understood that Brotli is a compressed format or

SetHandlerBrotliCompress(...)

Reply:
We have `CompressHandler` and `CompressHandlerLevel`. It would be super weird to call this similar method `SetHandlerBrotliLevel` all of sudden. It's better to keep names similar.